Doc/library/asyncio-tutorial/async-functions.rst

```diff
@@ -0,0 +1,60 @@
+Functions: Sync vs Async
+========================
+
+Regular Python functions are created with the keyword ``def``,
+and look like this:
+
+.. code-block:: python
+
+    def f(x, y):
+        print(x + y)
+
+    f(1, 2)
```

Comment:
Perhaps comment `# evaluate the function` above `f(1, 2)` and then remove the next sentence `The snippet...`